cms/templates/fragments/standalone-page-bootstrap.html

```diff
@@ -0,0 +1,27 @@
+## mako
+
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/studio-main.css" %>
```

Comment:
I introduced shared templates for rendering the fragment itself, one each for LMS and Studio.